.golangci.yml

```diff
@@ -38,6 +38,11 @@ linters-settings:
       - diagnostic
       - opinionated
       - performance
+    disabled-checks:
+      - unnamedResult
+      - hugeParam
```

Comment:
Awesome, I was unsure about `hugeParam` and whether that would imply making _all_ option structs pointers. I've updated the changelog to reflect this.